### PR TITLE
update lesson flow re: PATCH; fix grammar

### PR DESCRIPTION
**Pull request by gj**
- change intro to reflect that we've already been introduced to `UPDATE` in Rails and that most of this lesson deals with `DELETE`
- fix comma splices
- update Markdown link syntax

### CODE REVIEW

**Comment by gj** [comment]
I didn't want to remove it (b/c it's in the JavaScript section, which I'm woefully unqualified to be editing), but I imagine the extra `|` character here is unnecessary: https://github.com/learn-co-curriculum/delete-forms-rails/pull/6/files#diff-04c6e90faac2675aa89e2176d2eec7d8R124

**Comment by AnnJohn** [comment]
'cc @mendelB @PeterBell

File: README.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 So far, we've worked with two pieces of the CRUD puzzle:
```
Comment (PeterBell) [inline README.md]: Shouldn't this now be three pieces - not two?

**Comment by PeterBell** [comment]
LGTM other than the fact that it should be three items we've looked at - mind tweaking that @gj ?

**Comment by gj** [comment]
That's why they pay you the big bucks, @PeterBell. Fixed!

Also, what do you think about the JS question (above)?

**Comment by mendelB** [comment]
@gj I'll chime in here that the `|` does indeed look unnecessary - wanna add one more commit for that and I'll merge the pack?

**Comment by gj** [comment]
Done @mendelB

**Pull request merged**